src/libs/notifications/notifications.ts

```diff
@@ -62,16 +54,18 @@ export const navigateFromNotification = async (
 export const handleInitialNotification = async (
   client: CozyClient
 ): Promise<void> => {
-  const initialNotification = await messaging().getInitialNotification()
+  const notification = await messaging().getInitialNotification()
 
-  if (initialNotification) {
-    await navigateFromNotification(client, initialNotification)
+  if (notification?.data?.redirectLink) {
+    await navigateFromNotification(client, notification.data.redirectLink)
   }
 }
 
 export const handleNotificationOpening = (client: CozyClient): (() => void) => {
   return messaging().onNotificationOpenedApp(async notification => {
-    await navigateFromNotification(client, notification)
+    if (notification.data?.redirectLink) {
+      await navigateFromNotification(client, notification.data.redirectLink)
```

Comment:
we don't have to edit the jsdoc / type of this `navigateFromNotification` ?

Reply:
I updated the type in the first commit.

Reply:
Just a few lines above. Wondering why I didn't see it first..